src/diagnostics/netcdf_writer.jl

```diff
@@ -138,40 +141,48 @@ function target_coordinates(
     S <:
     Union{Spaces.CenterFiniteDifferenceSpace, Spaces.FaceFiniteDifferenceSpace},
 }
-    # Logarithmically spaced with base e
+    # Exponentially spaced with base e
+    #
+    # We mimic something that looks like pressure levels
+    #
+    # P ~ exp(-z/H)
+    #
+    # We assume H to be 7000, which is a good scale height for the Earth atmosphere
+    H_EARTH = 7000
 
     num_points_z = num_points[]
     FT = Spaces.undertype(space)
     vert_domain = space.topology.mesh.domain
     z_min, z_max = FT(vert_domain.coord_min.z), FT(vert_domain.coord_max.z)
-    # We are creating a log-space here, so we have to ensure that z_min > 0.
-    # If z_min = 0, we set the the first point to be at altitude of 100
-    log_z_min = z_min ≈ 0 ? FT(2) : log(z_min)
-    log_z_max = log(z_max)
-    return collect(exp.(range(log_z_min, log_z_max, num_points_z)))
+    # We add a floor of 100 meters to z_min to avoid having to deal with the singular value z = 0
+    z_min = z_min + 100
```

Comment:
How about `z_min = max(z_min, 30)` or `z_min = max(z_min, H_EARTH / 300)` in case we have non-zero `z_min` in the future? And I chose ~30 because for our target resolution we have three levels below 100m. I was thinking of using H_EARTH in case we ever change it, but that's up to you.

Reply:
Fixed, thanks